keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/model/ActiveServerGroup.kt

```diff
@@ -1,8 +1,10 @@
 package com.netflix.spinnaker.keel.clouddriver.model
 
 import com.fasterxml.jackson.annotation.JsonCreator
+import com.netflix.spinnaker.keel.model.Capacity
 import com.netflix.spinnaker.keel.model.Moniker
 
+// todo eb: this should be more general so that it works for all server groups, not just ec2
```

Comment:
Or we have different representations that Jackson can discriminate between?

Reply:
Ya that's tentatively what I was thinking.